HW1/P2/P2b.py

```diff
@@ -1,3 +1,27 @@
 from P2 import *
 
-# Your code here
+import findspark
+findspark.init('/home/toby/spark')
+
+import pyspark
+import numpy as np
+
+if __name__ == '__main__':
+    sc = pyspark.SparkContext(appName="Spark 2")
+    sc.setLogLevel('WARN') 
+
+    x = np.arange(1, 2001)
+    x = sc.parallelize(x)
+
+    pixels = x.cartesian(x)
+    pixels = pixels.partitionBy(100)
```

Comment:
Your P2b distribution looks like it has a very tight bound, which is good!  The graph is a little deceptive since its bimodal, but its definitely much closer than the original.